tests/typecheck/db/models/test_fields.yml

```diff
@@ -0,0 +1,159 @@
+-   case: db_models_fields_choices
```

Comment:
The `tests/assert_type/db/models/fields/test_choices.py` test is enough, no need to duplicate it. Can you please test invalid types here? At least one.

Reply:
I pushed an update now, was this what you had in mind?